packages/scrum-server/src/api/components/user/service.js

```diff
@@ -81,5 +82,24 @@ const localAuth = async req => {
     throw Error(e.message);
   }
 };
+const deleteProfileById = async userId => {
```

Comment:
these 2 functions bodies are the same, the only change is the query , I suggest refactor it

Reply:
Removi por el momento la otra, no se esta ocupando.